pack/ms.json

```diff
@@ -735,7 +776,7 @@
     "code": "33039",
     "deck_limit": 3,
     "faction_code": "jinteki",
-    "flavor": "Some tests require <strong>specific</strong> conditions.",
+    "flavor": "Some tests require specific conditions.",
```

Comment:
Am I wrong in thinking this flavour text is bolded, or is this an artefact of the source sheet missing that detail?

Reply:
no, it needs the bold.  I'll fix it.